.travis.yml

```diff
@@ -4,30 +4,16 @@ language: generic # Force travis to use its minimal image with default Python se
 compiler:
   - gcc
 env:
-  global:
-    - CATKIN_WS=~/catkin_ws
-    - CATKIN_WS_SRC=${CATKIN_WS}/src
-    - CI_ROS_DISTRO="indigo"
+  matrix:
+    - USE_DEB=true
+      ROS_DISTRO="kinetic"
+      ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - USE_DEB=true
+      ROS_DISTRO="kinetic"
+      ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+
 install:
-  - sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu trusty main" > /etc/apt/sources.list.d/ros-latest.list'
-  - wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq -y python-rosdep python-catkin-tools
-  - sudo rosdep init
-  - rosdep update
-  # Use rosdep to install all dependencies (including ROS itself)
-  - rosdep install --from-paths ./ -i -y --rosdistro $CI_ROS_DISTRO
+  - git clone https://github.com/davetcoleman/graph_msgs.git
```

Comment:
is it maybe because your scripts are running indigo and this is kinetic? kinetic has to run within a [Docker image](https://github.com/ros-industrial/industrial_ci/pull/52/files#diff-f9a62833538ab14fff849c2be16ab986R56) so maybe it can't see the cloned repo